core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -154,28 +155,34 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
   /**
     * The web app PUT the status of the given ([[UpdateSpec]], VIN) to [[UpdateStatus.Canceled]]
     */
-  def cancelUpdate(vin: Vehicle.Vin, uuid: Refined[String, Uuid]): Route = {
-    val response = db.run(UpdateSpecs.cancelUpdate(vin, uuid)).map {
-      i: Int => i match {
-          case 0 => HttpResponse(StatusCodes.BadRequest)
-          case _ => HttpResponse(StatusCodes.NoContent)
-        }
+  def cancelUpdate(ns: Namespace, vin: Vehicle.Vin, uuid: Refined[String, Uuid]): Route = {
```

Comment:
`UpdateSpecs.cancelUpdate()`, like all public methods in `UpdateSpecs`, returns `DBIO[...]`. That's the current design. This PR does its share of refactoring and boy scout rule, within the current design.